src/peoplemeasurement/migrations/0027_drop_redundant_views.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.7 on 2021-10-21 12:23
+
+from django.db import migrations
+
+from telcameras_v2.view_definitions import VIEW_STRINGS, get_view_strings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('peoplemeasurement', '0026_add_related_name_to_areas_and_lines_20211021_1135'),
+        ('telcameras_v2', '0032_deploy_v10_views_20210701_1411'),
+    ]
+
+    _view_names = [
+        "cmsa_15min_view_v7",
+        "cmsa_15min_view_v8",
+        "cmsa_15min_view_v9",
```

Comment:
Ik zie dat er nog referenties naar deze dropped tabellen staan in de volgende views...

- cmsa_15min_view_v7_realtime_predict
- cmsa_15min_view_v8_realtime_predict
- cmsa_15min_view_v8_realtime_predict_30d
- cmsa_15min_view_v9_realtime_predict
- cmsa_15min_view_v9_realtime_predict_30d